tools/oeedger8r/Emitter.ml

```diff
@@ -764,26 +778,33 @@ let gen_enclave_code (ec : enclave_content) (ep : edger8r_params) =
         fd.plist ]
     |> List.flatten
   in
-  let rec oe_gen_set_pointers prefix count setter (ptype, decl) =
+  let rec oe_gen_set_pointers parent prefix count setter (ptype, decl) =
     let size = oe_get_param_size (ptype, decl, "pargs_in->" ^ prefix) in
     let tystr = get_cast_to_mem_expr (ptype, decl) false in
     let arg = prefix ^ decl.identifier in
-    [ gen_c_for count
-    ; [sprintf "OE_%s_POINTER(%s, %s, %s);" (setter ptype) arg size tystr]
-    ; (let param_count =
-         oe_get_param_count (ptype, decl, "pargs_in->" ^ prefix)
-       in
-       flatten_map
-         (oe_gen_set_pointers
-            (arg ^ gen_c_deref param_count)
-            param_count setter)
-         (get_deepcopy_members (get_param_atype ptype))) ]
-    |> List.flatten
+    let check_expr =
+      (* TODO: Also gross. *)
```

Comment:
@anakrish This is just a code-style comment of something I want to clean up eventually.